src/client/app/+home/home.component.css

```diff
@@ -1,13 +1,25 @@
 :host {
   display: block;
-  padding: 0 16px;
-}
-
-input {
-  width: 250px;
+  background-color: #f0f0f0;
+  height: 100%;
+  margin-top: 0px;
+  padding: 20px 40px;
 }
 
 ul {
   list-style-type: none;
   padding: 0 0 0 8px;
 }
+
+li {
```

Comment:
Isn't it nice that we can use SUPER unspecific CSS rules like this without worrying that you're changing global rules for all <li> elements on the page?

(view encapsulation docs here)
https://angular.io/docs/ts/latest/guide/component-styles.html#!#view-encapsulation


Normally in CSS, this is a crazy bad code smell. It's un-performant and dangerous to apply style rules to ALL <li> elements. But thanks to Angular 2's view encapsulation, this actually should work just fine.